.parcelrc

```diff
@@ -0,0 +1,4 @@
+{
+  "extends": ["@parcel/config-default"],
+  "reporters":  ["parcel-reporter-static-files-copy"]
+}
```

Comment:
EOF

https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file